INSS.EIIR.Web/Constants/Common.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace INSS.EIIR.Web.Constants
+{
+    public static class INSOLVENCY_TYPE
+    {
+        public const string BANKRUPTCY = "Bankruptcy";
```

Comment:
can you change const to public static readonly

Reply:
Always use PascalCase for class names. Always use PascalCase for class names.

I  have asked to change to static member because its a static class.